### ISSUE TITLE
XML namespace URI mapped to wrong prefix for the xml:lang attribute

### ISSUE THREAD

**Issue by falko** [camunda/camunda-modeler#2214]
__Describe the Bug__

A roundtrip with the attached model changes the `xml:lang` attributes to `xml_1:lang` and defines a new prefix `xmlns:xml_1="http://www.w3.org/XML/1998/namespace"`.

The result is an invalid XML file and several other BPMN tools fail to open it. An XML schema validation shows:
```
Vacation Request.bpmn:2: namespace error : xml namespace URI mapped to wrong prefix
.org/2001/XMLSchema-instance" xmlns:xml_1="http://www.w3.org/XML/1998/namespace"
                                                                               ^
Vacation Request.bpmn:2: namespace error : Namespace prefix xml_1 for lang on definitions is not defined
1:lang="en" triso:translationLanguages="it,fr,de" trisobpmn:logoChoice="Default"
                                                                               ^
Vacation Request.bpmn:105: namespace error : Namespace prefix xml_1 for lang on translation is not defined
        <i18n:translation xml_1:lang="fr">Employé</i18n:translation>
                                         ^
Vacation Request.bpmn:106: namespace error : Namespace prefix xml_1 for lang on translation is not defined
        <i18n:translation xml_1:lang="de">Mitarbeiter</i18n:translation>
                                         ^
```

__Steps to Reproduce__

1. Open [Vacation Request.bpmn.txt](https://github.com/camunda/camunda-modeler/files/6311744/Vacation.Request.bpmn.txt)
2. Add a space to the name of the Pool "Employee" and remove it again (this should mark the model as changed)
3. Save the file under a new name

__Expected Behavior__

The namespace prefix of the `xml:lang` attribute is not changed and no extra namespace prefix is defined.

__Environment__

- OS: Ubuntu 20.04.2 LTS
- Camunda Modeler Version: 4.7.0 nightly build of 2021-04-14

The model uses the BPMN I18n specification: https://github.com/bpmn-miwg/bpmn-i18n
See also: https://www.w3.org/International/questions/qa-when-xmllang.en

----

This issue has been discovered during the preparation for the upcoming BPMN MIWG demo in December 2021 in Long Beach.

**Comment by nikku** [comment]
Thanks for reporting. I've already started to investigate this bug, cf. https://github.com/bpmn-io/moddle-xml/issues/60.

**Comment by nikku** [comment]
Fixed with [`bpmn-js@8.3.1`](https://github.com/bpmn-io/bpmn-js/blob/develop/CHANGELOG.md#831). Needs integration into Camunda Modeler.

**Comment by barmac** [comment]
Closed via https://github.com/camunda/camunda-modeler/pull/2246
